src/Sulu/Bundle/ContentBundle/Controller/SeoController.php

```diff
@@ -117,4 +117,12 @@ public function postAction(Request $request, $uuid)
             $this->view($result)
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getLocale(Request $request)
+    {
+        return $this->getLanguage($request);
```

Comment:
Since the `getLanguage` method is private anyway I would remove it, and move the code here.